readme.md

```diff
@@ -18,6 +18,7 @@ Multi-platform and multi-language.
 - Bot behavior is extremely customizable via configuration files (config as code)
 - Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
 - Lock files are natively supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
+- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement (npm packages only)
```

Comment:
This seems like a good place, it's near the end of the list items that cover dependencies, but before the items that talk about the supported platforms or methods of use.